docs/CONFIGURING-SLACK.md

```diff
@@ -26,63 +33,37 @@
  - users:read.email
 ```
 
-### User Token Scopes
-```
-- channels:read
-```
+## Slash Commands
 
-## Install the app to workspace
-- In __Features__ click on the __OAuth & Permissions__;
-- And click on the __Install App to Workspace__;
-- Select a channel for Slack to post as an app and click on the __Allow__;
-- Copy the __Bot User OAuth Access Token__;
-- Paste de code into the `HELLPER_OAUTH_TOKEN` variable;
-- Restart your application.
+- Now, go in __Features__, click on the __Slash Commands__ and click on the __Create New Command__ to add the following commands:
 
+| Command  | Request URL | Short Description |
+| - | - | - |
+|`/hellper_incident`|<https://yourhost.publicaddress.com/open>|_Starts Incident_|
+|`/hellper_status`|<https://yourhost.publicaddress.com/status>|_Show all pinned messages_|
+|`/hellper_close`|<https://yourhost.publicaddress.com/close>|_Closes Incident_|
+|`/hellper_resolve`|<https://yourhost.publicaddress.com/resolve>|_Resolves Incident_|
+|`/hellper_cancel`|<https://yourhost.publicaddress.com/cancel>|_Cancels Incident_|
+|`/hellper_pause_notify`|<https://yourhost.publicaddress.com/pause-notify>|_Pauses incident notification_|
+|`/hellper_update_dates`|<https://yourhost.publicaddress.com/dates>|_Updates the dates for an incident_|
 
-## Events
-- Now, in __Features__, click on the __Event Subscriptions__;
-- And in __Enable Events__ turn on it;
-- In __Request URL__, set your application's public URL to the field. It will look something like this: `https://yourhost.publicaddress.com/events`;
-- Under it you should receive a request that looks like this:
-```
-Our Request:
-POST
-"body": {
-	 "type": "url_verification",
-	 "token": "XXXXXXXX",
-	 "challenge": "YYYYYYY"
-}
-Your Response:
-"code": 200
-"error": "challenge_failed"
-"body": {
-
-}
-```
-_* you will receive the same info in your console application too._;
```

Comment:
Are we removing this? We should open a issue to remove the code related too.

Reply:
We no longer need the token because we are using Signing Secret. There is no related code. In fact this was never necessary, as the token is generated in the basic information of the application.

Reply:
I was talking about the Events Subscriptions, and the endpoint `/events`

Reply:
The endpoint is not being removed, we still need it. Only the way we use the token has changed.

Reply:
https://github.com/ResultadosDigitais/hellper/pull/129/files#diff-adbb3de4b2845b02408b80ffb779103cR54